configs/rollup.config.js

```diff
@@ -22,6 +22,7 @@ import commonjs from '@rollup/plugin-commonjs';
 export default {
   input: 'lib/cjs/bidiTab/bidiTab.js',
   output: {
+    name: 'mapperTab',
```

Comment:
do we need to access it? I think we do not?

Reply:
We do not need to access it, but the warning is noisy, the only intention of this commit is to silence it. I am also open to other ways of doing so